docker/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM rust:1.54 AS android-bindings-builder
+FROM rust:1.68 AS android-bindings-builder
 USER root
 ENV SDK_URL="https://dl.google.com/android/repository/commandlinetools-linux-8092744_latest.zip" \
     ANDROID_HOME="/usr/local/android-sdk" \
     ANDROID_VERSION=31 \
     ANDROID_BUILD_TOOLS_VERSION=32.0.0 \
+    NDK_VERSION="25.2.9519653" \
```

Comment:
Needed to update the ndk version because ndk-bundle is using 22.0, which doesn't provide libunwind for aarch64 targets.
The newer versions dropped the libunwind library and now use llvms unwind.